### ISSUE TITLE
vapoursynth no longer works with mpv due to unknown lenght clip

### ISSUE THREAD

**Issue by 4re** [mpv-player/mpv#1845]
Since https://github.com/vapoursynth/vapoursynth/commit/cc24d3fe7b67c805fe458678665cde3bd49698b6 vapoursynth throws an error due to unknown lenght clip.

```
        vf: Opening video filter: [vapoursynth file=~~/filters/gradfun3.py]
Unknown length (0) clips are no longer supported
Aborted
```

**Comment by wm4** [comment]
That's shitty.

**Comment by olifre** [comment]
Seems to be solved by a1e410e43a7d1cf638f9ce278ec8b7727ae9a32f , thanks wm4 !
I also agree that's a nonsense change on the VS side, especially since this totally breaks API compability.

**Comment by 4re** [comment]
Nice, thanks! :)
